scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -56,15 +56,24 @@ def get_machine_arch():
     return os.uname()[4]
 
 
-def get_os_loader():
-    path = '/usr/share/qemu/aavmf-aarch64-code.bin'
+def get_os_loader(firmware_type=""):
     if 'aarch64' in get_machine_arch():
-        return """
-  <loader readonly='yes' type='pflash'>%s</loader>
-""" % path
+        path = '/usr/share/qemu/aavmf-aarch64-code.bin'
+        return """<loader readonly='yes' type='pflash'>%s</loader>""" % path
+    elif 'x86_64' in get_machine_arch() and firmware_type == "uefi":
+        path = '/usr/share/qemu/ovmf-x86_64-ms-code.bin'
+        return """<loader readonly='yes' type='pflash'>%s</loader>""" % path
```

Comment:
Code duplication with line 62. Could be avoided by moving the return outside the if,
with something like
```python
if path=="":
   return ""
```
 coming first